defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+user_name       : "{{ azure_ad_user | default('ansible.test') }}"
```

Comment:
No need for default for unique items like username, password, client_id and secrets ... On the contrary they should be mandontary for  role